doc/topics/windows/windows-package-manager.rst

```diff
@@ -890,23 +877,19 @@ definition file:
         uninstaller: '{B5B5868F-23BA-297A-917D-0DF345TF5764}'
         uninstall_flags: '/qn /norestart'
         msiexec: True
-        cache_file: salt://win/repo/MyApp/MyApp.1.1.msp
-
-There are a few things to note about this software definition file. First, is
-the solution we are trying to solve, that of applying a patch. Version ``1.0``
-just installs the application using the ``1.0`` MSI defined in the ``installer``
-parameter. There is nothing special in the ``install_flags`` and nothing is
-cached.
+        cache_file: salt://win/repo-ng/MyApp/MyApp.1.1.msp
 
-Version ``1.1`` uses the same installer, but uses the ``cache_file`` option to
-specify a single file to cache. In order for this to work the MSP file needs to
-be in the same directory as the MSI file on the ``file_roots``.
+In the above example:
```

Comment:
Nitpick (style, non-blocking): Same comment as earlier. The Google Developer's style guide recommends using "previous" instead of "above" for accessibility reasons. See https://developers.google.com/style/accessibility and https://developers.google.com/style/ui-elements

Reply:
Updated